src/views/splash.html

```diff
@@ -43,18 +44,18 @@ <h2 id="splash-hack-cambridge-wordmark">Hack<br>Cambridge<br><span class="pixel"
                         <h4>
                             Social
                         </h4>
-                        <div>
-                            <a rel="external" href="https://www.facebook.com/hackcambridge">
-                                <img id="splash-facebook-logo" src="{{ asset('images/hc-2018/facebook-logo.svg') }}" alt="Facebook logo" draggable="false"/>
```

Comment:
Are we losing the alt text by making this change?  If we need to remove it, can we add similar accessibility information?

Reply:
I've changed the macro so that it now takes an icon name which may not be lower case and which is used for the SVG `<title>` (which the browser uses like alt text) and is then filtered with Nunjucks so it's lower case for the filename